src/actions/controller.ts

```diff
@@ -29,16 +29,38 @@ export class ActionsController {
     return ActionsController.actions.find(action => action.id === id);
   }
 
-  public static async performActionsFromParams() {
+  public static async performActionsFromURL() {
     const actionsParam = await getParameter("actions");
-    const actionsIds = uniq(compact(split(actionsParam, ",")));
-    getLogger().trace(`configuration - actionsIds= ${actionsIds}`, { method: "performActionsFromParams" });
+    getLogger().trace(`actionsParam= ${actionsParam}`, { method: "performActionsFromURL" });
+    if (actionsParam?.startsWith("%5B") || actionsParam?.startsWith("[")) {
+      ActionsController.perfomFullActions(actionsParam);
```

Comment:
I will try to think of a new name to emphasize this action is:
- defined on the fly.
- fully specified.
-  does not match an existing action

Reply:
maybe in-lined?

Reply:
renamed to perfomInlinedActions